package/gluon-respondd/luasrc/lib/gluon/upgrade/400-respondd-firewall

```diff
@@ -6,26 +6,46 @@ uci:delete('firewall', 'wan_announced')
 
 -- Allow respondd port on WAN to allow resolving neighbours over mesh-on-wan
 uci:section('firewall', 'rule', 'wan_respondd',
-  {
-    name = 'wan_respondd',
-    src = 'wan',
-    src_ip = 'fe80::/64',
-    dest_port = '1001',
-    proto = 'udp',
-    target = 'ACCEPT',
-  }
+{
+	name = 'wan_respondd',
+	src = 'wan',
+	src_ip = 'fe80::/64',
+	dest_port = '1001',
+	proto = 'udp',
+	target = 'ACCEPT',
+}
 )
 
 -- Restrict respondd queries to link-local addresses to prevent amplification attacks from outside
 uci:section('firewall', 'rule', 'client_respondd',
-  {
-    name = 'client_respondd',
-    src = 'client',
-    src_ip = '!fe80::/64',
-    dest_port = '1001',
-    proto = 'udp',
-    target = 'REJECT',
-  }
+{
+	name = 'client_respondd',
+	src = 'client',
+	src_ip = '!fe80::/64',
+	dest_port = '1001',
+	proto = 'udp',
+	target = 'REJECT',
+}
 )
 
+uci:section('firewall', 'rule',  'mesh_respondd_mcast1',
+{
+	name = 'mesh_respondd',
+	src = 'mesh',
+	src_ip = site.prefix6,
+	dest_port = '1001',
+	proto = 'udp',
+	target = 'ACCEPT',
+}
+)
+uci:section('firewall', 'rule',  'mesh_respondd_ll',
+{
+	name = 'mesh_respondd',
+	src = 'mesh',
+	src_ip = 'fe80::/64',
+	dest_port = '1001',
+	proto = 'udp',
+	target = 'ACCEPT',
+}
+)
```

Comment:
The ordering of commits is very weird. This adds 'mesh' rules (which didn't exist so far) in addition to 'client' rules, even though they aren't different zones in the final setup.

I think the first commit should introduce the 'drop' zone, move br-client to 'drop' by default, and have the batman-adv setup scripts move br-client to the existing client zone. After that, we would introduce the local_clients zone.
In the third commit, rename the 'client' zone to 'mesh' (while removing all rules for 'client' and recreating them with unchanged policy for 'mesh'), and then finally make the policy more restrictive in a fourth commit.